src/components/UserDropdownMenu/index.tsx

```diff
@@ -34,7 +34,7 @@ export const UserDropdownMenu = ({ handleOpenModal, setDropdownOpen }: UserDropd
               {button.title}
             </NavLink>
           ))}
-          <span className='cabinet-sidebar-nav_line' />
+          <div className='cabinet-sidebar-nav_line' />
```

Comment:
Why replace it with a div? It's better to make a decorative line with before/after